src/angular/styles/_typography.scss

```diff
@@ -650,6 +651,10 @@
     --sbb-datepicker-calendar-body-cell-padding: #{sbb.pxToRem(16.5)};
     --sbb-datepicker-calendar-body-cell-size: #{sbb.pxToRem(30)};
 
+    // Dialog
```

Comment:
The style for standard mode is currently broken. You need to define these variables on `:root` level, too.